pkg/openstackbaremetalset/baremetalhost.go

```diff
@@ -111,6 +111,8 @@ func BaremetalHostProvision(
 
 		if len(instance.Spec.DNSSearchDomains) > 0 {
 			templateParameters["CtlplaneDnsSearch"] = instance.Spec.DNSSearchDomains
+		} else {
+			templateParameters["CtlplaneDns"] = []string{}
```

Comment:
Just curious as to whether this should be `CtlplaneDnsSearch` instead of `CtlplaneDns`.